skll/config.py

```diff
@@ -390,7 +359,7 @@ def _parse_config_file(config_path, log_level=logging.INFO):
     do_grid_search : bool
         Whether to perform grid search.
     grid_objectives : list of str
-        A list of objects functions to use for tuning.
```

Comment:
Extremely nitpicky, but this should be "objective functions."

Reply:
Are you commenting on a line that was actually removed? :)

Reply:
Oh, my bad. I'm not sure how I did that! Please disregard. :)